catroid/src/main/java/org/catrobat/catroid/drone/jumpingsumo/JumpingSumoBrickFactory.java

```diff
@@ -20,28 +20,23 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.catrobat.catroid.content.bricks;
+package org.catrobat.catroid.drone.jumpingsumo;
 
-import android.view.View;
+import org.catrobat.catroid.content.bricks.BrickBaseType;
 
-import com.badlogic.gdx.scenes.scene2d.actions.SequenceAction;
+public final class JumpingSumoBrickFactory {
 
-import org.catrobat.catroid.R;
-import org.catrobat.catroid.content.Sprite;
-
-import java.util.List;
-
-public class DroneFlipBrick extends DroneBasicBrick {
-	private static final long serialVersionUID = 1L;
+	private JumpingSumoBrickFactory() {
+	}
 
-	@Override
-	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
-		sequence.addAction(sprite.getActionFactory().createDroneFlipAction());
-		return null;
+	public enum JumpingSumoBricks {
 	}
 
-	@Override
-	protected String getBrickLabel(View view) {
-		return view.getResources().getString(R.string.brick_drone_flip);
+	public static BrickBaseType getInstanceOfJumpingSumoBrick(JumpingSumoBricks brick) {
+
+		switch (brick) {
```

Comment:
this is innecessary

Reply:
what do you mean? I already said to you, that this is the file which is necessary for the bricks later. Here at the moment with no bricks, it has no function. But with the other tickets, it will be needed.

Reply:
ahja sry, i am a bit confused today